### PR TITLE
Update

### LINKED ISSUE #2: Add NPM version badge
```
[![NPM Version](https://img.shields.io/npm/v/json-nested-replace.svg)](https://www.npmjs.com/package/json-nested-replace)
```
Add this line in readme with other badges

### PR DESCRIPTION
**Pull request by ayanujju**
Added NPM Version badge

Fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Please consider donating, if you like my work
 ```
 $ npm install json-nested-replace
 ```
+## NPM Version Badge
```
Comment (arshadkazmi42) [inline README.md]: There is no need for this title.

File: README.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ Please consider donating, if you like my work
...
+[![NPM Version](https://img.shields.io/npm/v/json-nested-replace.svg)](https://www.npmjs.com/package/json-nested-replace)
```
Comment (arshadkazmi42) [inline README.md]: You can move this at the top of the file.
Right below the title and above `[![Build Status]`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Can you make these minor changes

File: README.md (hunk 1 above)
Comment (ayanujju) [reply README.md]: ok I'll remove it

File: README.md (hunk 2 above)
Comment (ayanujju) [reply README.md]: Done

**Comment by ayanujju** [comment]
Done with these all. Can you please review it

**Review by arshadkazmi42 — APPROVED** [review]
Great work. Thank you 🎉

**Comment by ayanujju** [comment]
Thanks

**Pull request merged**